docs/index.md

```diff
@@ -0,0 +1,30 @@
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) 
+for deploying and managing the [Chrony](https://chrony-project.org) NTP server 
+in your systems.
+
+This charm simplifies the configuration and maintenance of `chrony` across a 
+range of environments, enabling basic time provision and synchronization with 
+NTP servers.
+
+## Project and community
+
+The Chrony Operator is a member of the Ubuntu family. It's an
+open source project that warmly welcomes community projects, contributions,
+suggestions, fixes and constructive feedback.
+* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+* [Get support](https://discourse.charmhub.io/)
+* [Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)
+* [Contribute](https://charmhub.io/chrony/docs/contributing)
+* [Roadmap](https://charmhub.io/chrony/docs/roadmap)
+Thinking about using the Chrony charm for your next project? [Get in touch](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)!
+
+## Contributing to this documentation
+
+Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://discourse.charmhub.io/) to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to either directly change something you see that's wrong, or ask a question, or make a suggestion about a potential change via the comments section.
+
+If there's a particular area of documentation that you'd like to see that's missing, please [file a bug](https://github.com/canonical/penpot-operator/issues).
```

Comment:
Does this link need to be updated for chrony?

Reply:
Oops, updated, thanks!